### DISCUSSION TITLE
Accessing the markdown themes from a Vue component

### DISCUSSION THREAD

**Discussion by indaco** [vuejs/vitepress#3967] [Q&A]
Hi all, sorry for the thumb question.
I am trying to get access to the `markdown.themes` set in `.vitepress/config.js` from a custom Vue component.

it seems that the markdown configuration is not directly accessible through the `site.value` object provided by `useData`

How to do that?

thanks a lot for your help

**Comment by brc-dd** [comment]
What's your use case?

**Reply by indaco** [reply]
In my Vue component, I load a custom Shiki language to be used with  `getHighlighter` and `codeToHtml`.

So far, I have a `themes` prop exposed by the vue component, but I would like the theme to be inherited from the one set in the Vitepress config file.

 i do not know if there is a better way to do that.
 
 Thanks
